### PR TITLE
Correct minor typos in tournament logistics doc

### PR DESCRIPTION
**Pull request by thbsp**
I noticed a few typos when reading over this myself so I ran the whole thing through a grammar 
checker and made a few obvious corrections.

### CODE REVIEW

File: docs/guide/tournament-logistics.rst
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:466]: How about "Without revealing any details, try to get at least some guidance on how to pronounce unfamiliar names."?

File: docs/guide/tournament-logistics.rst
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:321]: Two options:
* "[...] you actually go to everyone of the venue [...]"
* "[...] you actually go to every venue [...]"

Splitting "everyone" isn't natural.

File: docs/guide/tournament-logistics.rst
Comment (tienne-B) [inline docs/guide/tournament-logistics.rst:115]: Neutral on this change; think "latter" can be used in relation to time (as used in the _Book of Mormon_ musical at least)

**Review by tienne-B — CHANGES_REQUESTED** [review]
Nice patch! Made me realize I'd never actually fully read that page! I've added a few comments that you could squash in.

As for the commit message, I would de-emphasize the "secret round" to "silent round", as that is also just a typo. How I make my commit messages is influenced by https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines (note the suggested max of 80 characters for the title and 100 characters for each line in the message). This commit could be amended for these comments/suggestions.

File: docs/guide/tournament-logistics.rst
Comment (czlee) [reply docs/guide/tournament-logistics.rst:321]: "every one of", as a synonym for "each of", is idiomatic ([Grammarly](https://www.grammarly.com/blog/everyone-vs-every-one/), [Writing Explained](https://writingexplained.org/everyone-vs-every-one-difference), [Lexico/Oxford](https://www.lexico.com/definition/every_one), [Collins](https://www.collinsdictionary.com/dictionary/english/every-one))

File: docs/guide/tournament-logistics.rst
Comment (czlee) [reply docs/guide/tournament-logistics.rst:115]: I think it's a typo and should be corrected to "later" 🙂

When used to refer to time, it has a connation of "as opposed to the earlier part of a period", _e.g._ "the latter part of the 20th century" or I guess "the latter part of the tournament", which doesn't apply in this context.

**Comment by czlee** [comment]
Just confirming that "secret round" is a typo and should be "silent round", thanks guys 🙂

**Comment by czlee** [comment]
Also, sharp eyes @thbsp, thanks for putting this PR in!

File: docs/guide/tournament-logistics.rst
Comment (thbsp) [reply docs/guide/tournament-logistics.rst]: Rephrased to "`- Without revealing any details, try to get some guidance on how to pronounce unfamiliar names.`"

**Comment by thbsp** [comment]
I edited the commit titles and messages as suggested.
(What do I do with the requested changes? I do personally 'resolve conversation' or leave that to one of the core members?)

**Review by tienne-B — APPROVED** [review]

**Comment by tienne-B** [comment]
Looks good! You may click on Resolve when you feel that the discussion is over (but I've done it already). I'll squash the update commit and reflow the message before merging though.

**Pull request merged**
